diff --git a/node_modules/@react-native-community/masked-view/android/build.gradle b/node_modules/@react-native-community/masked-view/android/build.gradle
index c4ee933..0bf598e 100644
--- a/node_modules/@react-native-community/masked-view/android/build.gradle
+++ b/node_modules/@react-native-community/masked-view/android/build.gradle
@@ -6,7 +6,7 @@ buildscript {
   if (project == rootProject) {
     repositories {
       google()
-      jcenter()
+      mavenCentral()
     }
 
     dependencies {
@@ -44,7 +44,7 @@ android {
 
 repositories {
   google()
-  jcenter()
+  mavenCentral()
   maven { url "https://jitpack.io" }
   maven {
     // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
